### ISSUE TITLE
Multiple Accounts/Token

### ISSUE THREAD

**Issue by bonxz** [lnxcz/discord-sniper#20]
Hey, it would be cool if you could use multiple tokens for sniping. Currently only one works or do I see it wrong?

**Comment by carlos3212345** [comment]
Yeh this feature sounds pretty nice to add into the bot.

**Comment by lnxcz** [comment]
Yes, now there's only support for one token. Hope I will release new version until febuary cuz i kinda stopped workin' on all my projects.
